### PR TITLE
use darklaf-core intellj as new look and feel

### PR DESCRIPTION
**Pull request by Seil0**
This pr adds darklaf-core, which adds support for multiple new look and feels.
Other changes:
* gradle 6.1.1 -> 6.3
* rework Configuration.java to use get value by key
* fix resource leak in TarBz2ZipUtil.java

### CODE REVIEW

File: viewer/src/main/java/rv/Globals.java [hunk 1]
```diff
@@ -2,8 +2,17 @@
 
 import java.awt.Image;
 import java.io.IOException;
+
```
Comment (Gama11) [inline viewer/src/main/java/rv/Globals.java:5]: Hm, are you sure you don't have Eclipse import sorting active accidentally? IIRC clang-format doesn't put empty lines between sections.

File: viewer/config.txt [hunk 2]
```diff
@@ -36,6 +36,7 @@ Drawing Port         : 32769
 General Settings:
 Record Logfiles      : false
 Logfile Directory    : 
+Look and Feel        : intellij
```
Comment (Gama11) [inline viewer/config.txt:39]: I still think a comment before this line with available theme names would be helpful so they're more discoverable. That would require tackling #86 though, I guess you don't want to do that during this PR?

Also, I wonder if darcula should be default.

Comment (Seil0) [reply viewer/config.txt:39]: I looked into #86 but it's not as easy to fix as i thought.

The default can be what ever we want.

File: viewer/src/main/java/rv/Globals.java (hunk 1 above)
Comment (Seil0) [reply viewer/src/main/java/rv/Globals.java]: fixed in aca3e21

File: viewer/config.txt (hunk 2 above)
Comment (Seil0) [reply viewer/config.txt:39]: I just asked the other members of team magma, we think light theme is the default for most applications. Although dark themes are on the rise. @Gama11 feel free to change the default. But we should definitely mention all possible themes in the read-me and change-log.

Comment (weisJ) [reply viewer/config.txt:39]: Side note: if you use darklaf `2.0.3` you use the operating system theme as the default.

**Comment by Gama11** [comment]
Respecting the system theme by default is perfect. :)

On Windows there's some annoying log output on startup, can we suppress this somehow? Not a blocker for merging though.

```
[2020-05-01 12:19:01] [INFO] Loaded darklaf-windows.dll. Native features are enabled. [at com.github.weisj.darklaf.platform.windows.WindowsLibrary]
```

**Comment by weisJ** [comment]
> On Windows there's some annoying log output on startup, can we suppress this somehow? Not a blocker for merging though.

Use `LafManager#enableLogging`. It may currently not work as expected but with version `2.1.1` It will be fixed and hide the log message appropriately.

**Comment by Gama11** [comment]
@weisJ Thanks!

**Pull request merged**